rm2/webpack.config.js

```diff
@@ -0,0 +1,31 @@
+//method one
+// module.exports = process.env.NODE_ENV === "development" 
+// ? require("./webpack.config.dev") : require("./webpack.config.prod");
```

Comment:
Consider not leaving commented code in the repo.
Additionally, consider going with 1 particular method for env specific webpack config.
Side note: I don't see how "method two" would chose between prod and dev configs, could you run me through the specifics?

Reply:
Yeah, I added the comments for learning purposes but I have removed it.
For the method two, i have 2 separate config for dev and prod respectively and reconfigured the package.json command to build and run for different env removing the method one